README.md

```diff
@@ -10,74 +10,140 @@ This project demonstrates how to build a modern note-taking application using Ne
 
 This demo is built with:
 
-- [Neon](https://neon.tech)
-- [Neon Auth](https://neon.tech/docs/guides/neon-auth)
-- Neon Data API (Postgrest-compatible)
+- [Neon](https://neon.tech) — Serverless Postgres
+- [Neon Auth](https://neon.com/docs/auth/overview) — Authentication with automatic JWT integration
+- [Neon Data API](https://neon.com/docs/data-api/get-started) — Direct database access from the frontend
+
+> **See it in action:** [Live Demo](https://neon-data-api-neon-auth.vercel.app/)
+>
+> **Tip:** If you encounter issues with social login providers, try email/password instead.
 
 ## Prerequisites
 
-Before you begin, ensure you have the following installed:
+Before you begin, ensure you have:
 
-- [Bun](https://bun.sh/) (v1.0 or newer)
-- A [Neon](https://neon.tech) account and project, with [Auth enabled](https://neon.tech/docs/guides/neon-auth)
+- [Bun](https://bun.sh/) (v1.0 or newer) installed
+- A [Neon account](https://console.neon.tech/signup) (free tier works)
 
 ## Getting Started
 
-Follow these steps to run the demo:
+### 1. Create a Neon Project with Auth and Data API
+
+1. Go to [pg.new](https://pg.new) to create a new Neon project
+2. In the Neon Console, navigate to your project and enable:
+   - **Neon Auth** — Go to the **Auth** page in the left sidebar and follow the setup wizard
+   - **Data API** — Go to the **Data API** page in the left sidebar and enable it
+
+For detailed instructions, see:
+
+- [Getting started with Neon Auth](https://neon.com/docs/auth/overview)
+- [Getting started with Data API](https://neon.com/docs/data-api/get-started)
+
+### 2. Clone and Install
+
+```bash
+git clone https://github.com/neondatabase-labs/neon-data-api-neon-auth.git
+cd neon-data-api-neon-auth
+bun install
+```
+
+### 3. Configure Environment Variables
+
+Create a `.env` file in the project root:
+
+```env
+# Neon Data API URL
+# Find this in Neon Console → Data API page → "Data API URL"
+VITE_NEON_DATA_API_URL=https://your-project-id.data-api.neon.tech
+
+# Neon Auth Base URL
+# Find this in Neon Console → Auth page → "Auth Base URL"
+VITE_NEON_AUTH_URL=https://your-project-id.auth.neon.tech
```

Comment:
fwiw my base URL structure shown in the console was `https://your-project-id.auth.neon.tech/neondb/auth` and I that's what I had to use in the .env file. Using this structure (i.e. without the full path) the demo didn't work: `Called reply with an invalid status code: 3D000` type errors. Same for Data API. Didn't dig into the details so not sure if it's just an issue on my machine